go.mod

```diff
@@ -1,35 +1,37 @@
 module github.com/openshift/cert-manager-operator
 
-go 1.21
+go 1.22.0
```

Comment:
This would require to update go version for any future security updates. It is better to make it work with any patch versions.